masterlist.yaml

```
@@ -2045,6 +2050,8 @@ plugins:
     msg:
       - <<: *patch3rdParty_KPH_CACO
         condition: 'active("Complete Alchemy & Cooking Overhaul.esp") and not active("CACO Rare Curios Patch.esp")  and not active("BSHeartland.esm")'
+      - <<: *patch3rdParty_ClockA
+        condition: 'active("Clockwork.esp") and not (active("ClockworkSorting_Ing_CCCurios.esp") or active("ClockworkSorting_Ing_CCCurios_CACO.esp"))'
```

Two `and not active("")` will work fine here since we only need to know if the one of the patches is active.

```suggestion
        condition: 'active("Clockwork.esp") and not active("ClockworkSorting_Ing_CCCurios.esp") and not active("ClockworkSorting_Ing_CCCurios_CACO.esp")'
```